cmd/sample-controller/main.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2022 The Clusternet Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	controller "github.com/clusternet/feedinventory-controller/pkg/controller"
+	workloadregistry "github.com/clusternet/feedinventory-controller/pkg/registry"
+
+	clusternet "github.com/clusternet/clusternet/pkg/generated/clientset/versioned"
+	informers "github.com/clusternet/clusternet/pkg/generated/informers/externalversions"
+	"github.com/clusternet/clusternet/pkg/known"
+
+	corev1 "k8s.io/api/core/v1"
+	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/controller-manager/pkg/clientbuilder"
+	"k8s.io/klog/v2"
+	"k8s.io/kubectl/pkg/scheme"
```

Comment:
I don't think we need to import `k8s.io/kubectl`.

Reply:
error import , will replace to "k8s.io/client-go/kubernetes/scheme"